### PR TITLE
Added karankartikey.md

### PR DESCRIPTION
**Pull request by karankartikeya**

### CODE REVIEW

File: june/16/karankartikeya.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Yesterday
+- I learned about various online editors like Glitch and Codepen
+- I commented on issue #27 in the nostalgify for my preview of the Navbar
+- I learned about the sticky navbar and fixed navbar
+
+# Today
+- I issued a PR to the User interface for adding the navbar page
```
Comment (YashKumarVerma) [inline june/16/karankartikeya.md:7]: use something like, sent a PR to user Interface to add navbar page.

File: june/16/karankartikeya.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Yesterday
+- I learned about various online editors like Glitch and Codepen
+- I commented on issue #27 in the nostalgify for my preview of the Navbar
+- I learned about the sticky navbar and fixed navbar
```
Comment (YashKumarVerma) [inline june/16/karankartikeya.md:4]: No need of using I again and again

File: june/16/karankartikeya.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+# Today
+- I issued a PR to the User interface for adding the navbar page
+- I will try to read the Backend scripts which have been merged
+
+# Blockers
+- Is it good to start reading the Backend scripts for learning `Backend` ?
+    -If not, then guide me how to start with it
```
Comment (YashKumarVerma) [inline june/16/karankartikeya.md:12]: Reply to all the blockers would be given after everyone has submitted their introspect.

**Review by YashKumarVerma — APPROVED** [review]

File: june/16/karankartikeya.md (hunk 1 above)
Comment (karankartikeya) [reply june/16/karankartikeya.md:7]: I will make sure of it from next time.

**Pull request merged**